stable/redis/templates/redis-master-svc.yaml

```diff
@@ -19,6 +19,12 @@ spec:
   {{ if eq .Values.master.service.type "LoadBalancer" -}} {{ if .Values.master.service.loadBalancerIP -}}
   loadBalancerIP: {{ .Values.master.service.loadBalancerIP }}
   {{ end -}}
+  {{if .Values.master.service.loadBalancerSourceRanges }}
+  loadBalancerSourceRanges:
+  {{ with .Values.master.service.loadBalancerSourceRanges }}
+{{ toYaml . | indent 4 }}
+  {{- end }}
+  {{- end }}
```

Comment:
Please apply the changes below:

```diff
- {{if .Values.master.service.loadBalancerSourceRanges }}
+ {{ if and (eq.Values.master.service.type  "LoadBalancer") .Values.master.service.loadBalancerSourceRanges }}
  loadBalancerSourceRanges:
- {{ with .Values.master.service.loadBalancerSourceRanges }}
+ {{- with .Values.master.service.loadBalancerSourceRanges }}
{{ toYaml . | nindent 4 }}
  {{- end }}
```

Apply the same suggestion to **redis-slave-svc.yaml** service too.

Reply:
Applied in 24b885d